src/main/java/duke/UI.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Encapsulates some of the output messages that will be shown to the user.
+ */
+public class UI {
```

Comment:
In the guidelines of CS2103T, abbreviations are written in the form of `Gui`, `Sql` and `Html`. Even though I agree that it's seemingly contradicting with Java's practices (`java.sql.SQLXML` and `java.util.UUID`), I think it might be better to stick to the module's coding standard.